pkg/registration/hub/manifests/rbac/managedcluster-clusterrolebinding.yaml

```diff
@@ -12,3 +12,6 @@ subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
   name: system:open-cluster-management:{{ .ManagedClusterName }}
+- kind: Group
```

Comment:
add some comments on the reasoning and plan for migration.

Reply:
We are adding a second group, as for aws flow we are trying to use [eks access entries](https://docs.aws.amazon.com/eks/latest/userguide/creating-access-entries.html) which doesn't allow us to allow access to group with system prefix "system".

Also based on kubernetes [docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects), "system" prefix is a reserved prefix for kubernetes internal usage.

The reason to add second group is to not update existing group abruptly and cause failure while upgrade existing installation to newer version of OCM. So, the plan is adding second group now to support aws flow and later in the next version, delete old group.

Reply:
Also request you to drive the migration of existing group for csr flow on your end.